File: backend/src/main/java/es/dawgrupo2/zendashop/basicDTO/OrderBasicDTO.java
package es.dawgrupo2.zendashop.basicDTO;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.LocalDateTime;

public record OrderBasicDTO(
    Long id,
    LocalDateTime creationDate,
    Boolean completed,
    BigDecimal totalPrice,
    UserBasicDTO user,
    String deliveryAddress,
    String deliveryNote,
    LocalDate deliveryDate
) {
    
}

File: frontend/app/dtos/OrderBasicDTO.ts
import type UserBasicDTO from "./UserBasicDTO";

export default interface OrderBasicDTO {
  id: number;
  creationDate: string;
  completed: boolean;
  totalPrice: number;
  user: UserBasicDTO;
  deliveryAddress: string;
  deliveryNote: string;
  deliveryDate: string;
}
